### DISCUSSION TITLE
Automations - Delay between loop iterations

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7752] [Help] [answered]
I have the below Automation 
<img width="568" alt="Screenshot 2022-09-13 at 1 14 06 PM" src="https://user-images.githubusercontent.com/80512170/189876070-855e2bff-3e06-4acb-8d5f-3f56ec7f57ff.png">

I need to have a delay between the loop iterations of 1000ms.
Query Rows
Loop Over Rows 
 - > Do something
 - > Sleep for 1s

Ideas ?

**Comment by melohagan** [comment, marked as answer]
I don't think this is currently possible - why do you need to do this?

**Reply by poirazis** [reply]
Hey @melohagan 
I need to do this to throttle the speed of sending SMS ( The DO Something ) step.

**Reply by melohagan** [reply]
Let's keep the discussion to your most recent one: https://github.com/Budibase/budibase/discussions/7753#discussioncomment-3642822
